prometheuspvesd/discovery.py

```diff
@@ -142,7 +142,7 @@ def _exclude(self, pve_list):
             if obj["status"] in map(str, self.config.config["exclude_state"]):
                 continue
 
-            if obj["vmid"] in map(str, self.config.config["exclude_vmid"]):
+            if str(obj["vmid"]) in self.config.config["exclude_vmid"]:
```

Comment:
IMO the map function is still required. What it does, is to cast every single entry a user added to the `exclude_vmid` option to a string. Otherwise the the list could hold strings or integers and the coparison might fail again. So in my opinion ensure `obj["vmid"]` to be a string **and** using the map would be the way to go.

Reply:
`_validate` throws a ValidationError with the message: 
````
101 is not of type 'string'

Failed validating 'type' in schema['properties']['exclude_vmid']['items']:
    {'type': 'string'}

On instance['exclude_vmid'][0]:
    101
```
That's why I assumed the call to map is redundant, but if integers should be allowed as well (which would make more sense in my opnion), the schema needs to be changed first

Reply:
Good point. Totally missed that, will try to fix that first.

Reply:
Ok, lets keep it this way :+1: